src/components/slider/slider.js

```diff
@@ -558,11 +560,13 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
      * @returns {*}
      */
     function percentToValue( percent ) {
-      return (min + percent * (max - min));
+      var adjustedPercent = reverse ? (1 - percent) : percent;
+      return (min + adjustedPercent * (max - min));
     }
 
     function valueToPercent( val ) {
-      return (val - min)/(max - min);
+      var percent = (val - min)/(max - min);
```

Comment:
Just one little thing I saw (nitpicky): The arithmetic operator should be spaced, just to be consistent.